### DISCUSSION TITLE
Can i join sir #Rom-builder-Team

### DISCUSSION THREAD

**Discussion by mevadadarshan19** [ROM-builders/temporary#17782] [General]
Hi sir. I have been already using ROM-builders and wants to join team.

My Trees
Device : https://github.com/mevadadarshan19/android_device_asus_X01BD.git
Common : https://github.com/mevadadarshan19/android_device_asus_sdm660-common.git
Kernel : https://github.com/ArrowOS-Devices/android_kernel_asus_X01BD.git
Vendor : https://github.com/khirendra03/proprietary_vendor_asus.git

Successful builds 
DerpFest : https://cirrus-ci.com/task/5450801447960576
Xtended : https://cirrus-ci.com/task/6111178439524352

Manifest : https://github.com/mevadadarshan19/local_manifest.git

**Comment by Apon77** [comment]
please follow #team instruction in telegram
